HHS/Head-Start-TTADP
# Add goal component to frontend

**Description of change**
The goals are only shown right now, they are not saved to the DB. Every new report has two fake goals added. In the future simply changing the "goals" field in default values to an empty array should allow goals loaded from the API to populate properly. The buttons on the goal component do not work. 

**How to test**
1) Pull down changes
2) Start a new activity report
3) Go to the `Goals and objectives` page. You should see two fake goals
4) Go to the review page. You should see two fake goals in the review section

Note the goals will not show up on an already saved activity report.

**Issue(s)**
* closes https://github.com/HHS/Head-Start-TTADP/issues/216

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
